src/Stacktrace.php

```diff
@@ -301,14 +301,18 @@ protected function stripPrefixFromFilePath(string $filePath): string
      */
     protected function getFrameArgumentsValues(array $frame): array
     {
-        if (!isset($frame['args'])) {
+        if (!isset($frame['args']) || empty($frame['args'])) {
```

Comment:
```suggestion
        if (empty($frame['args'])) {
```

The `empty` does also check for `isset` so it can be combined.

Reply:
Indeed :)